src/screens/crewStationScreen.cpp

```diff
@@ -65,6 +66,9 @@ CrewStationScreen::CrewStationScreen()
         });
     }
 #endif
+
+    // Initialize and play the impulse engine sound.
+    impulse_sound = std::unique_ptr<ImpulseSound>( new ImpulseSound(PreferencesManager::get("impulse_sound_enabled") == "1") );
```

Comment:
This is missing the default of "2". So when you open a crew station or the main screen per default on a fresh install, and never went to the options screen the engine sounds will be off. But if you first went to the options screen, the sound will be mainscreen only.